services/src/main/java/io/scalecube/services/ServiceRegistryImpl.java

```diff
@@ -115,7 +119,7 @@ private void loadMemberServices(DiscoveryType type, Member member) {
   /**
    * register a service instance at the cluster.
    */
-  public void registerService(Object serviceObject) {
+  public void registerService(Object serviceObject, Tag[] tags) {
```

Comment:
Why don't just pass here `Map<String, String> tags` is there any specific benefit for using separate Tag class and pass plain array?